pkg/cri/server/sandbox_run.go

```diff
@@ -326,6 +334,9 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		}
 
 		sandboxCreateNetworkTimer.UpdateSince(netStart)
+		span.AddEvent("finished pod network setup",
+			tracing.EventAttribute(tracing.Attribute("pod.network.setup.duration", time.Since(netStart).String())),
```

Comment:
Should we update `AddEvent` to accept the list of attributes? (I haven't seen other options passed), so it'll be:


```go
span.AddEvent("finished pod network setup", tracing.Attribute("pod.network.setup.duration", time.Since(netStart).String())
```

WDYT?

Reply:
Yes, I kept this so that we can pass `trace.EventOption` (which includes attributes, timestamps and stacktrace).  For our current use case we are only setting attributes, so I am fine just accepting attributes.